src/server/new-request-handlers/repos.js

```diff
@@ -1,12 +1,18 @@
 "use strict";
-var helper = require('../config/helpers.js');
+var github = require('../queries/github.js');
 var request = require('request');
+var users = require('../queries/users.js');
 
 module.exports = {
   handlePost : function(req, res, next) {
     var gitID = req.cookies.githubId;
-    helper.gitURL(gitID, req.body.link, function(nextPage){
-      res.json(nextPage);
-    });
+    users.getOneAsync(gitID)
+      .then(function(usrObj){
+        var token = usrObj.token;
+        github.queryAsync(req.body.link, usrObj.github_token)
```

Comment:
Write this in the promise style for consistency with the rest of the handlers:

.then(function(usrObj) {
  var token = usrObj.token;
  return github.queryAsync(req.body.link, usrObj.github_token); <-- this allows the next .then to contain the result from this call
}
.then(function(nextPage) {
  ..etc